### ISSUE TITLE
Gmail now showing "Message clipped" UI on (some) short emails

### ISSUE THREAD

**Issue by peterc** [hteumeuleu/email-bugs#113]
I haven't yet been able to debug what is triggering this, but a new development I've noticed is that Gmail is now giving the "Message clipped" interface on some shorter emails. For example, our 10KB subscription confirmation mail.

<img width="363" alt="Screenshot 2022-04-02 at 14 52 07" src="https://user-images.githubusercontent.com/118/161386361-e40efb32-3fdd-418a-8c3c-b59e72c58411.png">

This is occurring after all of the content of the mail (which is very short) and no extra content is revealed by following it. This template has been used for several years now without this effect occurring until this week.

I'm going to gradually remove elements from the mail until the issue goes away and will report back if I find out what specific element is triggering this Gmail bug.

Note that this is also triggering the equivalent "VIEW ENTIRE MESSAGE" button on iOS Gmail, so is not just limited to the Web client.

**Comment by peterc** [comment]
The plot thickens. If I send the very same email through Postmark instead of Sendgrid, no message clipped message. The bodies have identical code (almost - Postmark puts a different MIME part separator on and changes the charset on the MIME type for some reason, but these shouldn't affect clipping).

**Comment by peterc** [comment]
OK I might have resolved it. A slightly odd one this. In the last line of our email we were using a "middle dot" Unicode character in our street address. Removing this character has removed the issue from Gmail. Postmark's change to the character encoding changed how this character was encoded in quoted-printable from =B7 to =C2=B7. This one change appears to resolve the problem.

In short, I think Gmail's clipping can be triggered by specific Unicode characters..(?)

**Comment by revelt** [comment]
> In short, I think Gmail's clipping can be triggered by specific Unicode characters..(?)

yeah, it's been observed https://github.com/hteumeuleu/email-bugs/issues/41#issuecomment-462482515 — btw you can use https://detergent.io (from yours truly) to encode any text that's pasted into the HTML template, it also removes invisible characters and fixes English style

**Comment by peterc** [comment]
Ah thanks @revelt! I did search before but failed to dig into that one. Closing this as it doesn't appear to be a new issue, although it did not exist for this particular email (which is 7+ years old) even two weeks ago.. yet does now. Maybe the range of characters triggering this problem has shfited in some small way or SendGrid is automatically tweaking the email in some new way (since it is not a problem with Postmark).
